pallets/detach/src/lib.rs

```diff
@@ -575,14 +632,23 @@ pub mod pallet {
 			}
 		}
 
+		fn get_merkle_root(detach_collection: &DetachCollection) -> Hash256 {
+			let detach_hashes = detach_collection.get_abi_encoded_hashes();
+			// TODO Review - Should we sort the leaves?
+			// detach_hashes.sort_by(|a, b| Keccak256::hash(a).cmp(&Keccak256::hash(b)));
```

Comment:
can you check actual literature? it might be good?

Reply:
It makes sense since then the **user won't have to care about the order of the leaves** (although btw in the emitted event `CollectionDetached` we show the order). 

So it's not actually necessary or better, but can be more convenient. **I guess I'll just implement it then...**